packages/components/src/internal/form.ts

```diff
@@ -351,26 +355,24 @@ export class FormControlController implements ReactiveController {
   }
 
   /**
-   * Dispatches a non-bubbling, cancelable custom event of type `sl-invalid`.
-   * If the `sl-invalid` event will be cancelled then the original `invalid`
-   * event (which may have been passed as argument) will also be cancelled.
-   * If no original `invalid` event has been passed then the `sl-invalid`
+   * Dispatches a non-bubbling, cancelable custom event of type `sd-invalid`.
+   * If no original `invalid` event has been passed then the `sd-invalid`
    * event will be cancelled before being dispatched.
    */
   emitInvalidEvent(originalInvalidEvent?: Event) {
-    const slInvalidEvent = new CustomEvent<Record<PropertyKey, never>>('sd-invalid', {
+    // TODO: Reviewer, do you agree with this?  This prevents us from having to preventDefault() on the original event in every form element component.
```

Comment:
more specifically, event.preventDefault() can either happen here, or it needs to be implemented in every "handleInvalid" method to prevent the browser validation messages from showing up in their overlay style